### ISSUE TITLE
cache availability of FES

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1264]
This calls the FES one extra time before each request - which is excessive. Ok for now since it works, but in the future we should cache a boolean to mark whether FES is available or not. I'll file a separate issue.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/1261#discussion_r776762162_

**Comment by tomholub** [comment]
<img width="941" alt="image" src="https://user-images.githubusercontent.com/6306961/147776949-b023654e-10bd-4c0b-97be-cdf3e5d3fa07.png">

**Comment by ioanmo226** [comment]
@tomholub This seems outdated issue now.
Current code looks like this.

```
let email: String
var fesUrl: String? {
    get async throws {
        if helper == nil {
            self.helper = try await EnterpriseServerApiHelper(email: email)
        }

        return self.helper?.fesUrl
    }
}
....
guard let fesUrl = try await fesUrl else {
    throw EnterpriseServerApiError.noActiveFesUrl
}

```

**Comment by tomholub** [comment]
This issue, as originally defined, was I think fixed in https://github.com/FlowCrypt/flowcrypt-ios/issues/1467

However, It would be even better if this was checked only once per app startup (I think now it's once per class init. which may be only on app startup. so maybe we already did it). It would be for Roma to have a look at, later.

**Comment by ioanmo226** [comment]
ok
